### PR TITLE
Add resources for Shapes

### PR DESCRIPTION
**Pull request by Midn18**

### CODE REVIEW

File: src/main/java/com/app/edu/config/DataLoader.java [hunk 1]
```diff
@@ -193,10 +193,18 @@ CommandLineRunner populateData() {
             ));
 
             // shapes
-            shapeRepository.save(new ShapeEntity(1, "Cerc","src/main/resources/static/sounds/shapesSounds/circleSound.wav",
+            shapeRepository.save(new ShapeEntity(1, "Cerc", "src/main/resources/static/sounds/shapesSounds/circleSound.wav",
```
Comment (nevreless) [inline src/main/java/com/app/edu/config/DataLoader.java:196]: please change label to human language

Comment (MelissaD998) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: please don't make reviews after 5pm

Comment (Midn18) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: stfu

Comment (nevreless) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: prod urgent feature

Comment (MelissaD998) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: image

Comment (nevreless) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: moare pradactionu fara poze cu null pointer

Comment (MelissaD998) [reply src/main/java/com/app/edu/config/DataLoader.java:196]: gata cu fotosinteza, toata lumea la culcare

**Review by mariareee — APPROVED** [review]

**Pull request merged**